src/components/shared/playbackButton.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { graphql } from "gatsby"
+import { contentExists } from "../../utils/ug-utils"
+import Overlay from "./overlay"
+import Video from "./video"
+import { Portal } from "react-portal"
+
+import "../../styles/playbackButton.css"
+
+const PlaybackButton = ({ widgetData }) => {
+  const video = widgetData.relationships?.field_media_video
+  const videoId = video?.field_media_oembed_video.split("/").pop()
+  const videoType = video?.field_media_oembed_video.includes("vimeo")
+    ? "vimeo"
+    : "youtube"
+  const modalId = `modal-${widgetData.drupal_id}`
+
+  return contentExists(videoId) ? (
```

Comment:
Remove `contentExists` and use optional chaining instead to check null values. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining for more details